### DISCUSSION TITLE
What Bluetooth L.E does Xbox series x/s controller use?

### DISCUSSION THREAD

**Discussion by oldsheep0205** [ndeadly/MissionControl#731] [Q&A]
I have a xbox series x/s controller, how can i connect it to my switch?

**Comment by amurgshere** [comment]
Bluetooth LE is not supported yet according to the instructions, so XBox Series X/S controllers will not work AFAIK.

+1 for adding Bluetooth LE support though if it is, as I would really love to be able to use my XBox Series X/S controller.

**Reply by ndeadly** [reply]
Bluetooth LE support is an ongoing project. There are test builds available on my discord server
